src/controllers/update/update.manage.trusts.review.the.trust.controller.ts

```diff
@@ -1,31 +1,48 @@
 import { NextFunction, Request, Response } from 'express';
-
+import { getApplicationData } from '../../utils/application.data';
+import { postTrustDetails } from "../../utils/trust.details";
 import { logger } from '../../utils/logger';
-import {
-  UPDATE_MANAGE_TRUSTS_INTERRUPT_URL,
-  UPDATE_MANAGE_TRUSTS_REVIEW_FORMER_BO_URL,
-  UPDATE_MANAGE_TRUSTS_REVIEW_THE_TRUST_PAGE,
-} from '../../config';
+import { getBoIndividualAssignableToTrust, getBoOtherAssignableToTrust } from '../../utils/trusts';
+import * as mapperDetails from '../../utils/trust/details.mapper';
+import * as mapperBo from '../../utils/trust/beneficial.owner.mapper';
+import { UPDATE_MANAGE_TRUSTS_INTERRUPT_URL, UPDATE_MANAGE_TRUSTS_REVIEW_THE_TRUST_PAGE } from '../../config';
+import { beginTrustReview } from '../../utils/update/review_trusts';
 
 export const get = (req: Request, res: Response, next: NextFunction) => {
   try {
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
 
+    const appData = getApplicationData(req.session);
+    
+    // if reviewing a trust, mark trust as in review
+    beginTrustReview(appData);
```

Comment:
As per the comment on the interrupt controller, this needs moved back to the post method after the submitted update to the trust has been validated/saved to app data.